db/migrate/20220326111340_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.text :Text
+      t.text :CommentsCounter
+      t.text :LikesCounter
+      t.references :Author, foregin_key: { to_table: 'users' }, index: true
```

Comment:
- [ ] Kindly note that it's always a good practice to name all columns in a Postgres table in lowercase and separate words with `_`. example `comments_counter` instead of `CommentsCounter`, `text` instead of `Text`.